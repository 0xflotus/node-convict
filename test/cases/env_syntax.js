exports.conf = {
  foo: {
    bar: {
      format: 'string ["a", "b"]',
      env: "BAR"
    }
  }
};

exports.env = {
  BAR: 'c' // hey!  that's not an allowable value!
};
