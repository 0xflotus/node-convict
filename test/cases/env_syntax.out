foo.bar: Instance is not one of the possible values: ["a","b"]
